Services/include/housekeeping/housekeeping_service.h

```diff
@@ -28,9 +28,12 @@
 #include "adcs.h"
 #include "eps.h"
 #include "housekeeping_athena.h"
+#include "housekeeping_charon.h"
+#include "housekeeping_athena.h"
 #include "hyperion.h"
 #include "sband.h"
 #include "uhf.h"
+//#include "ex2_hal/ex2_payload_software/hardware_interface/include/payload.h"
```

Comment:
This path should be in the include path

Reply:
This file doesn't even exist. @yunqingyi any idea where this came from?

Reply:
Waiting on this to merge

Reply:
I'm going to get rid of it